File: CS/DashboardXpoDemo/DataAccess/RevenueAnalysisDataItem.cs
using System;
using DevExpress.Xpo;

namespace DashboardXpoDemo.DataAccess {
    public class RevenueAnalysisDataItem : XPObject {
        public RevenueAnalysisDataItem(Session session) : base(session) { }
        int fYear;
        public int Year {
            get { return fYear; }
            set { SetPropertyValue(nameof(Year), ref fYear, value); }
        }
        string fState;
        public string State {
            get { return fState; }
            set { SetPropertyValue(nameof(State), ref fState, value); }
        }
        string fCategory;
        public string Category {
            get { return fCategory; }
            set { SetPropertyValue(nameof(Category), ref fCategory, value); }
        }
        string fProduct;
        public string Product {
            get { return fProduct; }
            set { SetPropertyValue(nameof(Product), ref fProduct, value); }
        }
        decimal fRevenue;
        public decimal Revenue {
            get { return fRevenue; }
            set { SetPropertyValue(nameof(Revenue), ref fRevenue, value); }
        }
        int fUnitsSold;
        public int UnitsSold {
            get { return fUnitsSold; }
            set { SetPropertyValue(nameof(UnitsSold), ref fUnitsSold, value); }
        }
    }
}

File: VB/DashboardXpoDemo/DataAccess/RevenueAnalysisDataItem.vb
Imports System
Imports DevExpress.Xpo

Namespace DashboardXpoDemo.DataAccess
	Public Class RevenueAnalysisDataItem
		Inherits XPObject

		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub
		Private fYear As Integer
		Public Property Year() As Integer
			Get
				Return fYear
			End Get
			Set(ByVal value As Integer)
				SetPropertyValue(NameOf(Year), fYear, value)
			End Set
		End Property
		Private fState As String
		Public Property State() As String
			Get
				Return fState
			End Get
			Set(ByVal value As String)
				SetPropertyValue(NameOf(State), fState, value)
			End Set
		End Property
		Private fCategory As String
		Public Property Category() As String
			Get
				Return fCategory
			End Get
			Set(ByVal value As String)
				SetPropertyValue(NameOf(Category), fCategory, value)
			End Set
		End Property
		Private fProduct As String
		Public Property Product() As String
			Get
				Return fProduct
			End Get
			Set(ByVal value As String)
				SetPropertyValue(NameOf(Product), fProduct, value)
			End Set
		End Property
		Private fRevenue As Decimal
		Public Property Revenue() As Decimal
			Get
				Return fRevenue
			End Get
			Set(ByVal value As Decimal)
				SetPropertyValue(NameOf(Revenue), fRevenue, value)
			End Set
		End Property
		Private fUnitsSold As Integer
		Public Property UnitsSold() As Integer
			Get
				Return fUnitsSold
			End Get
			Set(ByVal value As Integer)
				SetPropertyValue(NameOf(UnitsSold), fUnitsSold, value)
			End Set
		End Property
	End Class
End Namespace